org.freedesktop.Sdk.Extension.golang.appdata.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
this file cannot be YAML. It must be XML.

Reply:
Well, what I mean more precisely is, yes it *can* be YAML per https://www.freedesktop.org/software/appstream/docs/sect-AppStream-YAML.html but I am genuinely unsurprised that support for that hasn't been tested.

Reply:
Thanks for the comment. Where do I file a bug for that issue? At flatpak-builder or somewhere else?

I'll revert the changes soon as any issue/pull request will take longer.